src/request.ts

```diff
@@ -807,6 +807,9 @@ class DatastoreRequest {
    * included in *all* indexes, you must supply an `excludeFromIndexes` array.
    * See below for an example.
    *
+   * To avoid automatically large properties to indexed, you can supply an `excludeLargeProperties`
+   * as true or false. By default it is *false*.
+   *
```

Comment:
Maybe something like

```
To prevent large properties from being included in *all* indexes, you must pass
`excludeLargeProperties: true`
```
And place it above `* See below for an example.` line.

Reply:
Done, Thanks